api/python/provisioner/commands/__init__.py

```diff
@@ -431,6 +438,34 @@ def run(self, *args, targets=ALL_MINIONS, dry_run=False, **kwargs):
 
         return self._run(params, targets)
 
+@attr.s(auto_attribs=True)
+class PillarDump(CommandParserFillerMixin):
+    input_type: Type[inputs.PillarKeysList] = inputs.PillarKeysList
+    _run_args_type = RunArgsPillarDump
+
+    @classmethod
+    def from_spec(
+        cls, input_type: str = 'PillarKeysList'
+    ):
+        return cls(input_type=getattr(inputs, input_type))
+
+
+    def run(
+        self, *args, targets: str = ALL_MINIONS, local: bool = False,
+        **kwargs
+    ):
+        res = PillarGet.run(self)
+        os.makedirs(f"{PRVSNR_CORTX_CONFIG_DIR}", exist_ok=True)
+
+        resp_loaded = json.loads(json.dumps(res).replace('null', '""'))
+        for key in resp_loaded.keys():
+            resp_loaded[key] = ({k:v
```

Comment:
you may drop need here additional parenthesis here

Reply:
unwanted_keys = ("mine_functions", "provisioner", "glusterfs")
```
for key in resp_loaded.keys():
    if key in unwanted_keys:
        del resp_loaded[key]
```

Reply:
@ypise Addressed to make it more user-friendly. Thanks.